src/api/helpers/map-inactive-applications/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Application } from '.keystone/types'; // eslint-disable-line
+import { APPLICATION } from '../../constants';
+import { CronApplicationInactiveUpdate } from '../../types';
+
+/**
+ * mapInactiveApplications
+ * generates array of objects for updating the db
+ * object contains where object with id of application to be updated
+ * contains data object with fields which are updated
+ * @param {Array<CronApplicationInactiveUpdate>} applications: array of application ids and statuses
+ * @returns {Array<CronApplicationInactiveUpdate>}: Array of objects for updating db
+ */
+const mapInactiveApplications = (applications: Array<Application>) => {
+  /**
+   * loops through applications array
+   * adds id to where object
+   * adds abandoned status to status and previous status
+   * pushes to updateArray
+   */
+  const updateArray = applications.map((application: Application) => {
+    const update = {
+      where: { id: application.id },
+      data: {
+        status: APPLICATION.STATUS.ABANDONED,
+        previousStatus: application.status,
+        updatedAt: new Date(),
+      },
+    } as CronApplicationInactiveUpdate;
+
+    return update;
+  });
+
+  return updateArray;
```

Comment:
Not a blocker, can be done after this PR, i'd use `mapped` instead of `update` since it's not actually updating anything, is purely mapping